usmqe_tests/web/tendrl/test_login.py

```diff
@@ -0,0 +1,89 @@
+"""
+Description: Simple tests for log in on tendrl UI
+
+Author: ltrilety
+"""
+
+
+import pytest
+from selenium.webdriver.common.keys import Keys
+
+from webstr.selenium.ui.support import WaitForWebstrPage
+from usmqe.web.tendrl.loginpage import pages as loginpage
+from usmqe.web.tendrl.mainpage.landing_page.pages import get_landing_page
+
+LOGGER = pytest.get_logger('login_test', module=True)
+
+
+def test_positive_login(testcase_set, testcase_end):
```

Comment:
Well, even if I don't like it much, it doesn't close the door for putting tests together. In other words one tests do something and the other one starts exactly where the previous one ended. As I said I'd like more individual tests which does not require any other tests to run before they run, but it could be needed in future.